pyglmnet/utils.py

```diff
@@ -130,3 +135,22 @@ def set_log_level(verbose):
             raise ValueError('verbose must be of a valid type')
         verbose = logging_types[verbose]
     logger.setLevel(verbose)
+
+
+def return_verbose_iterable(data):
+    """
+    Wrap an iterable object with tqdm. If tqdm is not available
+    or if we did not set the appropriate log level, then we
+    fall back to the classical method
+    :param data: it can be a list or range() object
+    :param verbose: the verbosity level
+    :return: wrapped object
```

Comment:
can you document the docstring in a more standard way? You know how it should be :)

Reply:
Updated. I will need to change my PyCharm autocompletion to match this one :)